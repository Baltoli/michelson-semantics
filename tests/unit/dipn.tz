$Instructions { DIP 3 { PUSH bool True } } ; 
$Stack ListItem(5) ListItem(4) ListItem(3) ListItem(2) ListItem(1) ListItem(0) ;
