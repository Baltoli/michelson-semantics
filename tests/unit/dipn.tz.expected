<generatedTop>
  <k>
    .
  </k>
  <stack>
    5 ~> 4 ~> 3 ~> true ~> 2 ~> 1 ~> 0
  </stack>
  <paramtype>
    #NotSet
  </paramtype>
  <storagetype>
    #NotSet
  </storagetype>
  <balance>
    #Mutez ( 0 )
  </balance>
  <amount>
    #Mutez ( 0 )
  </amount>
  <now>
    #Timestamp ( -1 )
  </now>
  <myaddr>
    #Address ( -1 )
  </myaddr>
  <knownaddrs>
    .Map
  </knownaddrs>
  <sourceaddr>
    #Address ( -2 )
  </sourceaddr>
  <senderaddr>
    #Address ( -3 )
  </senderaddr>
  <chainid>
    #ChainId ( 0 )
  </chainid>
</generatedTop>
